File: 백준/Bronze/2010. 플러그/플러그.cc
#include <iostream>

using namespace std;

int main()
{
    int multitapCount;
    int plugCount;
    int accessablePC = 0;

    cin >> multitapCount;

    for (int i = 0; i < multitapCount; i++) {
        cin >> plugCount;

        accessablePC = accessablePC + plugCount;
    }

    accessablePC = accessablePC - (multitapCount - 1);

    cout << accessablePC;

    return 0;
}

File: 백준/Bronze/2010. 플러그/플러그.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int multitapCount;
        int plugCount;
        int accessablePC = 0;

        multitapCount = sc.nextInt();

        for (int i = 0; i < multitapCount; i++) {
            plugCount = sc.nextInt();

            accessablePC = accessablePC + plugCount;
        }

        accessablePC = accessablePC - (multitapCount - 1);

        System.out.print(accessablePC);
    }
}
